docs/__tests__/__snapshots__/sitemap.test.ts.snap

```diff
@@ -1,166 +1,43 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Sitemap Snapshot 1`] = `
-"/, 
-/android, 
```

Comment:
looks like upgrading glob broke our sitemap script?

Reply:
Actually I don't think this is related to this change specifically, not sure what caused it but I noticed the sitemap snapshot was failing in a few other docs prs